Description: Disable Jakarta Slide WebDAV library
 Jakarta Slide WebDAV library is not available in Debian,
 and this project was retired by its upstream developers.
 Please see http://jakarta.apache.org/slide/.
Author: Miguel Landaeta <miguel@miguel.cc>
Forwarded: not-needed
Last-Update: 2011-05-10

--- gradle-1.0~m3.orig/subprojects/core/core.gradle
+++ gradle-1.0~m3/subprojects/core/core.gradle
@@ -51,7 +51,7 @@ dependencies {
             "commons-codec:commons-codec:1.5@jar",
             libraries.guava,
             "commons-collections:commons-collections3:3.2.1@jar",
-            "slide:webdavlib:2.0@jar",
+            //"slide:webdavlib:2.0@jar",
             libraries.asm_all,
             'org.fusesource.jansi:jansi:1.4',
             'org.jruby.ext.posix:jna-posix:1.0.1'
--- gradle-1.0~m3.orig/subprojects/core/src/main/groovy/org/gradle/api/internal/artifacts/repositories/WebdavRepository.java
+++ gradle-1.0~m3/subprojects/core/src/main/groovy/org/gradle/api/internal/artifacts/repositories/WebdavRepository.java
@@ -17,7 +17,6 @@ package org.gradle.api.internal.artifact
 
 import org.apache.commons.httpclient.HttpsURL;
 import org.apache.ivy.plugins.repository.url.URLRepository;
-import org.apache.webdav.lib.WebdavResource;
 
 import java.io.File;
 import java.io.IOException;
@@ -52,9 +51,12 @@ public class WebdavRepository extends UR
         String destinationFileName =  destination.substring(fileNameStart + 1);
         HttpsURL hrl = new HttpsURL(baseUrl);
         hrl.setUserinfo(user, userPassword);
+        /*
         WebdavResource wdr = new WebdavResource(hrl);
         wdr.putMethod(wdr.getPath() + '/' + destinationFileName, source);
         wdr.close();
+        */
+        throw new UnsupportedOperationException("put");
     }
 
     //    Alternative implementation with httpclient only. Unfortunately this is slower.
